pkg/log/slackReporter.go

```diff
@@ -0,0 +1,79 @@
+package log
+
+import (
+	"bytes"
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"runtime"
+	"strings"
+
+	"github.com/google/uuid"
+	"golang.org/x/sys/unix"
+)
+
+const (
+	reportURL = "https://us-central1-tensorleap-ops3.cloudfunctions.net/demo-contact-bot"
+)
+
+var cloudMessageUuid string
+
+type MessageLevel string
+
+const (
+	InfoLog    MessageLevel = "info"
+	WarningLog MessageLevel = "warning"
+	ErrorLog   MessageLevel = "error"
+)
+
+func SendCloudReport(messageLevel MessageLevel, messageType string, payload *map[string]interface{}) error {
```

Comment:
You never return an error on this function so you can remove the return error

Reply:
Updated